### ISSUE TITLE
Wonky colorcodes in messages.json

### ISSUE THREAD

**Issue by ghost** [arthurbambou/Fabric---Discord-Link#133]
**Discription**
Not sure if i'm missing something or if it's a bug. I'm experiencing problems with assigning colorcodes in messages.json for "discordToMinecraft."
This is how I've got it setup:
`  "discordToMinecraft": {
    "message": "§9\u003c%player\u003e§r %message"
  }`
This works just like I'd expect when turning on the server and testing. When I start the server for a second time however, the code gets modified to this:
`  "discordToMinecraft": {
    "message": "?9\u003c%player\u003e?r %message"
  }`
For some reason § gets turned to "?"

**Reproduction**
1. Change `"discordToMinecraft": {
    "message": "[%player] %message"
  }` in fdlink.json to `  "discordToMinecraft": {
    "message": "§9\u003c%player\u003e§r %message"
  }`
2. Run server > stop server.
3. Run server again.
4. Check code in fdlink.json.

**Expected behaviour**
The code should remain unchanged after reboot.

**Additional context**
- Minecraft version: 1.16.5
- Fabric-Discord Link version: 0.8.9
- A list of all mods:
 -- fabric-api-0.29.4+1.16
 -- Fabric-Discord-Link-0.8.9

Was unsuccessful in pasting .json files here, so:
- note: Nothing has been modified except for channel ids, webhook, token and "discordToMinecraft" message.
<details> 
  <summary><strong>fdlink.json</strong></summary>
```
{
  "chatChannels": [
    "EDITED OUT"
  ],
  "logChannels": [
    "EDITED OUT"
  ],
  "webhookURL": "EDITED OUT",
  "ignoreBots": true,
  "activityUpdateInterval": 120,
  "minecraftToDiscord": {
    "general": {
      "enableDebugLogs": false
    },
    "chatChannels": {
      "commandPrefix": "-",
      "allowDiscordCommands": false,
      "serverStartingMessage": true,
      "serverStartMessage": true,
      "serverStopMessage": true,
      "serverStoppingMessage": true,
      "customChannelDescription": false,
      "minecraftToDiscordTag": false,
      "minecraftToDiscordDiscriminator": false,
      "playerMessages": true,
      "teamPlayerMessages": true,
      "joinAndLeaveMessages": true,
      "advancementMessages": true,
      "challengeMessages": true,
      "goalMessages": true,
      "deathMessages": true,
      "sendMeCommand": true,
      "sendSayCommand": true,
      "adminMessages": false,
      "atATellRaw": false,
      "achievementMessages": true
    },
    "logChannels": {
      "serverStartingMessage": true,
      "serverStartMessage": true,
      "serverStopMessage": true,
      "serverStoppingMessage": true,
      "customChannelDescription": false,
      "minecraftToDiscordTag": false,
      "minecraftToDiscordDiscriminator": false,
      "playerMessages": false,
      "teamPlayerMessages": false,
      "joinAndLeaveMessages": true,
      "advancementMessages": false,
      "challengeMessages": false,
      "goalMessages": false,
      "deathMessages": true,
      "sendMeCommand": true,
      "sendSayCommand": true,
      "adminMessages": true,
      "atATellRaw": false,
      "achievementMessages": true
    }
  },
  "discordToMinecraft": {
    "pingLongVersion": false
  },
  "emojiMap": [
    {
      "name": "example_name",
      "id": ":example_id:22222222"
    },
    {
      "name": "example_name2",
      "id": ":example_id2:22222222"
    }
  ],
  "version": 8,
  "token": "EDITED OUT"
}
```

</details>

<details> 
  <summary><strong>messages.json</strong></summary>
``````
{
  "minecraftToDiscord": {
    "playerMessage": {
      "customMessage": "\u003c%player\u003e %message",
      "useCustomMessage": true
    },
    "teamPlayerMessage": {
      "customMessage": "%team \u003c%player\u003e %message",
      "useCustomMessage": true
    },
    "serverStarting": "Server is starting!",
    "serverStarted": "Server Started.",
    "serverStopping": "Server is stopping!",
    "serverStopped": "Server Stopped.",
    "channelDescription": "Playercount : %playercount/%maxplayercount,\n Uptime : %uptime",
    "playerJoined": {
      "customMessage": "%player joined the game",
      "useCustomMessage": true
    },
    "playerJoinedRenamed": {
      "customMessage": "%new (formerly known as %old) joined the game",
      "useCustomMessage": true
    },
    "playerLeft": {
      "customMessage": "%player left the game",
      "useCustomMessage": true
    },
    "advancementTask": {
      "customMessage": "%player has made the advancement %advancement",
      "useCustomMessage": true
    },
    "advancementChallenge": {
      "customMessage": "%player has completed the challenge %advancement",
      "useCustomMessage": true
    },
    "advancementGoal": {
      "customMessage": "%player has reached the goal %advancement",
      "useCustomMessage": true
    },
    "deathMsgPrefix": "",
    "deathMsgPostfix": "",
    "atATellRaw": "%message",
    "meMessage": {
      "customMessage": "* %author %message",
      "useCustomMessage": true
    },
    "adminMessage": {
      "customMessage": "[%author: %message]",
      "useCustomMessage": true
    },
    "sayMessage": {
      "customMessage": "[%author] %message",
      "useCustomMessage": true
    },
    "achievement": {
      "customMessage": "%player has just earned the achievement %achievement",
      "useCustomMessage": true
    }
  },
  "discordToMinecraft": {
    "message": "§9\u003c%player\u003e§r %message"
  },
  "discord": {
    "commandPrefix": "!",
    "botActivities": [
      "!commands",
      "%playercount / %maxplayercount",
      "on %ip",
      "%uptime_D day(s), %uptime_H hour(s), %uptime_M minute(s) and %uptime_S second(s)"
    ]
  }
}
```
</details>

**Comment by R6219** [comment]
Experiencing the same issue still in latest version; well detailed here. Tried using `\u00A7` instead of `§` and both were replaced by question marks upon the servers' second restart. Could be due to host's file manager system-- we are using Pebblehost budget. 

Have others been able to successfully use colored and formatted text in "discordToMinecraft" messages?

**Comment by thecatcore** [comment]
Will be fixed in 0.10.1.
